tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -511,15 +512,25 @@ async def test_train_tensorboard_logging(
         {
             "pipeline": [
                 {"name": "WhitespaceTokenizer"},
-                {"name": "CountVectorsFeaturizer"},
+                {
+                    "name": "CountVectorsFeaturizer",
+                    "analyzer": "char_wb",
+                    "min_ngram": 3,
+                    "max_ngram": 17,
+                    "max_features": 10,
+                    "min_df": 5,
+                },
                 {
                     "name": "DIETClassifier",
                     EPOCHS: 1,
                     TENSORBOARD_LOG_LEVEL: log_level,
                     TENSORBOARD_LOG_DIR: str(tensorboard_log_dir),
+                    MODEL_CONFIDENCE: "linear_norm",
+                    CONSTRAIN_SIMILARITIES: True,
                     EVAL_NUM_EXAMPLES: 15,
                     EVAL_NUM_EPOCHS: 1,
                 },
+                {"name": "EntitySynonymMapper"},
```

Comment:
we can skip this component, no? It doesn't add anything to the test

Reply:
we can. there will be a warning but we can for sure skip that